packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -1223,20 +1220,14 @@ class _ContextMenuRouteStaticState extends State<_ContextMenuRouteStatic> with T
   }
 
   Alignment _getChildAlignment(Orientation orientation, _ContextMenuLocation contextMenuLocation) {
-    switch (contextMenuLocation) {
-      case _ContextMenuLocation.center:
-        return orientation == Orientation.portrait
-          ? Alignment.bottomCenter
-          : Alignment.topRight;
-      case _ContextMenuLocation.right:
-        return orientation == Orientation.portrait
-          ? Alignment.bottomCenter
-          : Alignment.topLeft;
-      case _ContextMenuLocation.left:
-        return orientation == Orientation.portrait
-          ? Alignment.bottomCenter
-          : Alignment.topRight;
-    }
+    return switch ((contextMenuLocation, orientation)) {
```

Comment:
nit: would it be more obvious that portrait is special if it's written this way?
```dart
(center || left || right, portrait) => Alignment.bottomCenter, 
 (_ContextMenuLocation.center, Orientation.landscape) => ...,
```

Reply:
Just refactored it to use an if-statement. Hopefully it's more readable now!